src/t8_cmesh/t8_cmesh_examples.h

```diff
@@ -101,14 +101,37 @@ t8_cmesh_t          t8_cmesh_new_hypercube (t8_eclass_t eclass,
                                             int do_bcast, int do_partition,
                                             int periodic);
 
+/** Construct a hypercube forest from one primitive tree class.
```

Comment:
To describe the coordinates in general is a little bit difficult, because the hypercube can contain diffrent distorted trees, if one chooses the boundary coordinates accordingly. You may see this, if you run the abouve example.
But I have added a small simple example to the documentation and hope that this makes thinks clearer.

Reply:
I renamed to `t8_cmesh_new_hypercube_pad`. This fits the purpose even better in my opinion.

Reply:
Sounds good to me!